rasa/core/policies/policy.py

```diff
@@ -292,6 +292,23 @@ def _default_predictions(domain: Domain) -> List[float]:
 
         return [0.0] * domain.num_actions
 
+    def format_tracker_states(self, states: List[Dict]) -> Text:
+        """Format tracker states to human readable format on debug log
+
+        Args: 
+            states: list of tracker states dicts
+        Returns:
+            the string of the states with user intents and actions
+        """
+        formatted_states = [""]
```

Comment:
why do we need empty string here?
```suggestion
        formatted_states = []
```

Reply:
Because the string returned by the function **format_tracker_states**  should begin with a line break , as it returns a join on `\n`, this empty string will guarantee that each state, even the first, will be displayed in a single line, with the following format:

```python
DEBUG rasa.core.policies.memoization  - Current tracker state:
[state 1] previous action: action_listen | user intent: greet
[state 2] previous action: utter_greet | user intent: greet
```

Instead, with an empty array to initialize the variable, the line breaks would only exist between the tracker states, resulting on the following: 

```python
DEBUG rasa.core.policies.memoization  - Current tracker state: [state 1] previous action: action_listen | user intent: greet
[state 2] previous action: utter_greet | user intent: greet
```

Reply:
could you please add brief comment to the code